addons/br/nfse/extensions.go

```diff
@@ -16,6 +17,22 @@ const (
 )
 
 var extensions = []*cbc.KeyDefinition{
+	{
+		Key: ExtKeyCNAE,
+		Name: i18n.String{
+			i18n.EN: "CNAE code",
+			i18n.PT: "Código CNAE",
+		},
+		Desc: i18n.String{
+			i18n.EN: here.Doc(`
+				The CNAE (National Classification of Economic Activities) code for a service.
+
+				For further details on the list of possible codes, see:
+
+				* https://www.ibge.gov.br/en/statistics/technical-documents/statistical-lists-and-classifications/17245-national-classification-of-economic-activities.html
+			`),
+		},
```

Comment:
Is there a pattern that could added here? Some kind of validation would be nice.

Reply:
Yes, thanks! I was going to add it. 

BTW, I opened in draft mode because it's completely work in progress. There are other changes coming. I'll let you know when is ready for review.

Reply:
Addressed in 08d41a80a1bf55e11cfefa4abde6e12790c7f553